.ci/cico_rhche_prcheck.sh

```diff
@@ -43,7 +43,8 @@ CHE_VERSION=$(getVersionFromPom)
 export CHE_VERSION
 
 echo "Running ${JOB_NAME} PR: #${RH_PULL_REQUEST_ID}, build number #${BUILD_NUMBER} for che-version:${CHE_VERSION}"
-.ci/cico_build_deploy_test_rhche.sh
+#.ci/cico_build_deploy_test_rhche.sh
```

Comment:
why this got commented?

Reply:
To speed up the test to the part we added, to see if it works. Removed now.